ts/screens/main/MessagesScreen.tsx

```diff
@@ -67,26 +88,70 @@ class MessagesScreen extends React.Component<Props, never> {
     ) : null;
   };
 
-  private renderMockedMessages = (
+  public getOrganizationName = (senderServiceId: string): string => {
```

Comment:
getOrganizationName is not a good name becouse you are returning "organizationName + department_name". Also pass both value as props separately.

Reply:
Fixed, i have split serviceOrganizationName in : serviceOrganizationName and serviceDepartmentName.